### PR TITLE
Make way for HTML transcriptions

### PR DESCRIPTION
**Pull request by commuterjoy**
Moving some mappings around,

```
brightcove.description -> standfirst
brightcove.long_description -> alternativeStandfirsts.promotionalStandfirst
videoEditor.transcription.html -> bodyHTML
```

### CODE REVIEW

File: server/models/video.js [hunk 1]
```diff
@@ -26,11 +27,11 @@ function video ({ capiV2, brightcove, videoEditor }) {
 
 	model.set('alternativeTitles', normalV2.alternativeTitles || {});
 
-	model.set('standfirst', brightcove.metadata.description);
+	model.set('standfirst', brightcove.metadata.long_description);
 
 	model.set('byline', normalV2.byline);
 
-	model.set('bodyHTML', `<p>${brightcove.metadata.long_description}</p>`);
+	model.set('bodyHTML', extractTranscription(videoEditor));
```
Comment (i-like-robots) [inline server/models/video.js]: This will need a fallback as content is rejected without a body.

**Comment by i-like-robots** [comment]
Does the base branch for this PR need changing?

**Comment by commuterjoy** [comment]
@i-like-robots ok, i think this is right

File: server/models/video.js [hunk 2]
```diff
@@ -28,9 +29,13 @@ function video ({ capiV2, brightcove, videoEditor }) {
 
 	model.set('standfirst', brightcove.metadata.description);
 
+	model.set('alternativeStandfirsts', {
+		promotionalStandfirst: brightcove.metadata.long_description || {}
```
Comment (i-like-robots) [inline server/models/video.js]: `... || undefined` might work better because this gets turned into a JSON string

Comment (commuterjoy) [inline server/models/video.js]: ok boss. done.

**Pull request merged**